### ISSUE TITLE
Error: no valid primary key self-signature

### ISSUE THREAD

**Issue by laserlemon** [keybase/keybase-issues#2172]
![screen shot 2016-04-01 at 12 03 41 pm](https://cloud.githubusercontent.com/assets/34264/14212634/db7d85fe-f801-11e5-9bc4-68395e8d3de2.png)

I can't encrypt when trying to send `indirect` a message but I can encrypt for `bryckbost`. Help, please! Thank you!

**Comment by maxtaco** [comment]
Yeah, that public key expired 2015-07-03.. We need a better error message there, and potentially a checkbox to allow you to encrypt anyway

**Comment by laserlemon** [comment]
I'm not totally security-savvy, so how did you get at the expiration date of the key? Also, why would I be able to encrypt for one recipient but not another?

**Comment by maxtaco** [comment]
It's not your key that expired, it's `indirect`'s key that expired

You can import his key into GPG and it will tell you

```
curl https://keybase.io/indirect/key.asc | gpg --import
gpg --edit-key A089D900
```

clunky, I know..

**Comment by rodrigograca31** [comment]
This is still an issue. I thought it was my key.... this error message should report the date or tell us its the other person key...
